app/src/main/java/edu/uc/app/parkfinder/service/ParkService.kt

```diff
@@ -22,6 +23,7 @@ class ParkService {
              */
             override fun onFailure(call: Call<ArrayList<Park>>, t: Throwable) {
                 // TODO: fill this in
+                Log.e( "ArrayList Park", "Error Occurred!", t);
```

Comment:
Added an error logged on failure

Reply:
This partially duplicates a previous review.

Previous review used Log.d, while Log.e is a better choice here.